### PR TITLE
Add sql-test-utill Module

### PR DESCRIPTION
**Pull request by SasinduDilshara**
>In order to test the `Processor` classes in the `sql.nativeimpl` package, we need the java interface of the connector modules. 
`sql-test-utils` module will be the java interface for `sql test client`.

### CODE REVIEW

File: sql-ballerina/Ballerina.toml [hunk 1]
```diff
@@ -20,4 +27,4 @@ path = "./lib/hsqldb-2.5.1.jar"
 groupId = "org.hsqldb"
 scope = "testOnly"
 artifactId = "hsqldb"
-version = "2.5.1"
+version = "2.5.1"
```
Comment (niveathika) [inline sql-ballerina/Ballerina.toml:30]:
```suggestion
version = "2.5.1"

```

**Review by niveathika — COMMENTED** [review]
@SasinduDilshara Could you send the PRs for JDBC & MySQL and tag them here in the description?

File: sql-ballerina/Ballerina.toml (hunk 1 above)
Comment (SasinduDilshara) [reply sql-ballerina/Ballerina.toml:30]: I will do it

**Comment by SasinduDilshara** [comment]
> @SasinduDilshara Could you send the PRs for JDBC & MySQL and tag them here in the description?

@niveathika akka, https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/85 this is the PR for JDBC module.

**Comment by trajinthan** [comment]
> > In order to test the `Processor` classes in the `sql.nativeimpl` package, we need the java interface of the connector modules.
> > `sql-test-utils` module will be the java interface for `sql test client`.

[MySQL Migration PR ](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/80)

**Pull request merged**